### PR TITLE
Theorems for cosmic and related properties

### PR DESCRIPTION
**Pull request by prabau**
Various theorems regarding cosmic and related properties.  This implements what was discussed in #511.

Similarly to [P182 ](https://topology.pi-base.org/properties/P000182/)(countable network) which was introduced to better manage P74 (cosmic), I have introduced [P183 ](https://topology.pi-base.org/properties/P000183/)(countable $k$-network) corresponding to P179 ($\aleph_0$-space).  This makes things more convenient as countable $k$-network is implied by [second countable] and also by [anticompact + countable].

I did not do it, but one could also introduce the properties [has a $\sigma$-locally finite network] and [has a $\sigma$-locally finite $k$-network] to parallel $\sigma$-space and $\aleph$-space.  Could be done at some later point if it ends up being convenient.  Feel free to comment.

### CODE REVIEW

**Review by StevenClontz — COMMENTED** [review]
This is just a comment.

File: theorems/T000011.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+uid: T000011
+if:
+  P000183: true
+then:
+  P000182: true
+---
+
+Evident from the definitions, as a $k$-network is a network.
```
Comment (StevenClontz) [inline theorems/T000011.md:9]: This comment points out this specific line.

**Review by akiraiwasa — APPROVED** [review]
having a countable k-network implies having a countable network

**Comment by prabau** [comment]
Thanks for the review.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**